fontforgeexe/collabclientui.h

```diff
@@ -165,7 +165,11 @@ extern int Collab_getLastChangedPos( void );
 extern int Collab_getLastChangedCodePoint( void );
 
 
-typedef void (*collabclient_notification_cb)( FontViewBase* fv );
+/**
+ * See https://developer.gnome.org/gobject/stable/signal.html
+ * for implied parameters in GLib callback functions.
+ */
```

Comment:
It was just being done wrong. Since I don't think many use the collab feature, it's never been fully highlighted as an issue I guess.